modules/visual_script/visual_script_builtin_funcs.h

```diff
@@ -67,6 +67,7 @@ class VisualScriptBuiltinFunc : public VisualScriptNode {
 		MATH_LERP,
 		MATH_INVERSE_LERP,
 		MATH_RANGE_LERP,
+		MATH_SMOOTH_STEP,
```

Comment:
I believe this breaks compatibility if put here: it should be put at the end of the list. You can see in the doc changes that all constants got new values, so if someone is using a function from below, suddenly the functions will be changed and it'll break the code.

Reply:
I just remember the time I've implemented wrapi/wrapf function and it was ok to putted them in the middle of that list.. but ok I will place it on the bottom

Reply:
Done !